perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -126,6 +126,17 @@ sub ward : Path : Args(2) {
     my @wards = split /\|/, $ward || "";
     $c->forward( 'body_check', [ $body ] );
 
+    # If viewing multiple wards, rewrite the url from
+    # /reports/Borsetshire?ward=North&ward=East
+    # to
+    # /reports/Borsetshire/North|East
+    my @ward_params = $c->get_param_list('ward');
+    if ( @ward_params ) {
+        $c->stash->{ward_names} = [ @wards, @ward_params ];
```

Comment:
Any reason you didn't use `wards` here - it looks like you do want to pass them through short_name for escaping purposes, given it's the names coming through?